tests/integration/server/courseInstance/courseInstance.service.test.ts

```diff
@@ -199,6 +201,53 @@ describe('Course Instance Service', function () {
         });
       });
     });
+    describe('Ordering', function () {
+      it('should order by area => prefix => integer => letter', function () {
```

Comment:
I would suggest maybe splitting these out into 4 separate tests if possible to make sure that each level of sorting works the way it should on it's own. 

1. Insert some courses into the DB with different areas, make sure the areas are all in the correct order
2. Insert some courses all with the same area, make sure the course prefixes are ordered correctly
3. Insert some courses with the same area and prefix, make sure that the integers are in the correct order
4. Finally, insert some courses with the same area, prefix and integer and make sure the letters are all in in the correct order.

Reply:
Fair enough - that makes sense.